db-sql/fake-sql/faker.js

```diff
@@ -0,0 +1,33 @@
+const faker = require('faker');
+
+// The following generates one single fake datapoint.
+
+module.exports = () => {
+  const result = {
+    place_name: `${faker.company.companyName()}`,
+    formatted_address: `${faker.address.streetAddress()}, ${faker.address.city()}, ${faker.address.stateAbbr()} ${faker.address.zipCode()}, ${faker.address.countryCode()}`,
+    international_phone_number: `${faker.phone.phoneNumber()}`,
+    email: `${faker.internet.email()}`,
+    website: `${faker.internet.url()}`,
+    open_now: faker.random.boolean(),
+    mon_open: Math.ceil(Math.random()*12),
```

Comment:
Although this is relatively simple, do you think it might be useful to create a simple helper function that does both Math.ceil(math.random*12)? This may help you incase you decide to change your logic later and keep code DRY ("don't repeat yourself")